### PR TITLE
Updated "Windows Server Update Services (WSUS)" section & Support timeline

### PR DESCRIPTION
**Pull request by jasonstephen15**
**Rendered articles for review:**

Managing WebView2 applications
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/enterprise?branch=pr-en-us-1568

Introduction to Microsoft Edge WebView2
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/?branch=pr-en-us-1568

### CODE REVIEW

File: microsoft-edge/webview2/concepts/enterprise.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ IT admins can use group policy objects (GPO) to configure policy settings for We
 
 
 <!-- ====================================================================== -->
+## Windows Server Update Services (WSUS)
+
+[Windows Server Update Services (WSUS)](/windows-server/administration/windows-server-update-services/get-started/windows-server-update-services-wsus) enables information technology administrators to deploy the latest Microsoft product updates. You can use WSUS to fully manage the distribution of updates of WebView2 that are released through Microsoft Update to computers on your network.
+
+> [!NOTE]
+> The recommended way of receiving WebView2 updates is by using the default Edge updater. Any modification of update & servicing paths should be done with caution.
```
Comment (vchapel) [inline microsoft-edge/webview2/concepts/enterprise.md:36]:
```suggestion
> The recommended way of receiving WebView2 updates is by using the default Microsoft Edge updater. Any modification of update & servicing paths should be done with caution.
```

Comment (vchapel) [reply microsoft-edge/webview2/concepts/enterprise.md:36]: Also, should "Microsoft Edge updater" be a link? Is that the official name? I couldn't find a good link ...

**Review by champnic — APPROVED** [review]
These changes on their own look fine. I wonder if we should also link to the WSUS deployment info from the "Distributing" doc.

File: microsoft-edge/webview2/concepts/enterprise.md [hunk 2]
```diff
@@ -28,6 +28,18 @@ IT admins can use group policy objects (GPO) to configure policy settings for We
...
+
+### WebView2 Deployment & Update using WSUS
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/enterprise.md:38]:
```suggestion
### WebView2 Deployment and Update using WSUS
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

File: microsoft-edge/webview2/concepts/enterprise.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/enterprise.md:36]:
```suggestion
> The recommended way of receiving WebView2 updates is by using the default Edge updater. Any modification of update and servicing paths should be done with caution.
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

File: microsoft-edge/webview2/concepts/enterprise.md [hunk 3]
```diff
@@ -28,6 +28,18 @@ IT admins can use group policy objects (GPO) to configure policy settings for We
...
+
+WebView2 options in Configuration Manager exist under the Microsoft Edge Management node. For more information, visit [Update Microsoft Edge](/mem/configmgr/apps/deploy-use/deploy-edge)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/enterprise.md:40]:
```suggestion
WebView2 options in Configuration Manager exist under the Microsoft Edge Management node. For more information, visit [Update Microsoft Edge](/mem/configmgr/apps/deploy-use/deploy-edge).
```

File: microsoft-edge/webview2/index.md [hunk 4]
```diff
@@ -136,7 +138,7 @@ WebView2 apps can run on the following versions of Windows.
 *   Windows Server 2008 R2 \*\*
 
 > [!IMPORTANT]
-> \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
+> WebView2 support for Windows 7 and Windows Server 2008 R2 will have the same support timeline as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
```
Comment (mikehoffms) [inline microsoft-edge/webview2/index.md:141]: Removing ** here might not work, because ** remains, two lines above, at "Windows Server 2008 R2 \*\*".

Comment (mikehoffms) [reply microsoft-edge/webview2/index.md:141]: Generally, usually, when possible, style guides for dev docs say to try to use present tense.  This spot could be an exception where future tense is required/appropriate.
https://docs.microsoft.com/en-us/style-guide/grammar/verbs#verb-tense
https://developers.google.com/style/timeless-documentation

Comment (mikehoffms) [reply microsoft-edge/webview2/index.md:141]: The word 'navigate' doesn't appear in MWSG.  The word 'see' is standard for "Click this provided link and read the article that immediately appears (no multi-step navigation/wayfinding/course-plotting is required)."
```suggestion
> WebView2 support for Windows 7 and Windows Server 2008 R2 will have the same support timeline as Microsoft Edge.  For more information, see [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/accessibility-terms

**Review by champnic — APPROVED** [review]

**Review by mikehoffms — APPROVED** [review]

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**